nexus/cmd/nexus/main.go

```diff
@@ -17,7 +18,8 @@ func main() {
 
 	flag.Parse()
 
-	server.SetupLogger(true)
```

Comment:
why not use another flag?

Reply:
don't mind either way...

Reply:
flag is hard to use since it is launched from python.. i think this is fine for now.